dockermodelrunner/models/api.go

```diff
@@ -0,0 +1,62 @@
+package models
+
+import "github.com/docker/docker-sdk-go/dockermodelrunner/modeldistribution/types"
+
+// ModelCreateRequest represents a model create request. It is designed to
```

Comment:
question: this sounds all fine but it's used to "pull" models which is confusing. Should this be changed?

Reply:
I think we chose using ModelCreate for adding future operations in the create namespace. Reading the original docs:
```
// ModelCreateRequest represents a model create request. It is designed to
// follow Docker Engine API conventions, most closely following the request
// associated with POST /images/create. At the moment is only designed to
// facilitate pulls, though in the future it may facilitate model building and
// refinement (such as fine tuning, quantization, or distillation).
```

I like the idea of using `ModelPullRequest` although, after reading that, I'm concerned if we would be stopping ourselves for doing more things in that

Reply:
As user I would expect pull and create to be seperate, much like docker build and docker pull are seperate.

It would be good to confirm the thinking behind using create before making a call.